xblockutils/helpers.py

```diff
@@ -0,0 +1,17 @@
+"""
+Useful helper methods
+"""
+
+
+def child_isinstance(block, child_id, block_class_or_mixin):
```

Comment:
It's not unreasonable, just something perhaps to keep in the back of your mind as the codebase evolves.